python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/vrfs.schema.yml

```diff
@@ -27,6 +27,9 @@ keys:
           type: bool
         ip_routing_ipv6_interfaces:
           type: bool
-        tenant:
-          type: str
-          description: Key only used for documentation or validation purposes.
+        metadata:
+          type: dict
```

Comment:
```suggestion
          type: dict
          description: |-
            The data under `metadata` is used for documentation, validation or integration purposes.
            It will not affect the generated EOS configuration.

```